packages/RichTextEditing-Core.package/TextStructure.class/instance/isKern.st

```diff
@@ -0,0 +1,4 @@
+testing
+isKern
+
+	^ self attributes anySatisfy: [:each | each isKern]
```

Comment:
What does it do?

Reply:
`TextKern` is a Attribute that influences letter spacing. `isKern` is also implemented by `TextAttribute` so it is overwritten to reflect the attributes of the TextStructure.